test/02-investigate.t

```diff
@@ -255,6 +267,34 @@ has "$got" 'WORKER_CLASS:vim=foo,duh,bar' "job assigns the WORKER_CLASS only to
 try clone 10021 10030 last_good_tests_and_build:123 refspec
 has "$got" 'unidentified worker class in vars.json' "info shown in the name is WORKER_CLASS is empty"
 
+curl() {
+    if [[ "$@" =~ investigation_ajax ]]; then
+        echo -n '{"diff_packages_to_last_good": "Diff of packages not available",
+              "diff_sut_packages_to_last_good": "Diff of packages not available",
```

Comment:
Are all these lines necessary?

Reply:
no all of them. But I want to represent the $investigation as it is expected.

Reply:
@perlpunk can we close this?

Reply:
No, please only simulate the parts necessary for the specific tests to make tests robust and easier to understand. There is no good benefit to simulate everything in every test

Reply:
When the question is "is this necessary?" and the answer is no, then it's likely that it should be removed :)